modules/opta-k8s-service-helm/templates/deployment.yaml

```diff
@@ -64,18 +64,13 @@ spec:
             - name: {{ .name | quote }}
               value: {{ .value | quote }}
             {{ end }}
-            {{ range $val := .Values.linkSecrets }}
-            - name: {{ $val.name | quote }}
-              valueFrom:
-                secretKeyRef:
-                  name: secret
-                  key: {{ $val.name | quote }}
-                  optional: true
-            {{ end }}
           envFrom:
           - secretRef:
               name: manual-secrets
               optional: true
+          - secretRef:
+              name: secrets
+              optional: true
```

Comment:
correct me if I'm wrong here:
`opta secret` would only updates `manual-secrets` and not change `secrets` (which contains the linked secret)

but since `envFrom: secretRef:secrets` is defined last, it will take precedence, so the container would use the old value at runtime.

_envFrom: When a key exists in multiple sources, the value associated with the last source will take precedence._ 
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#container-v1-core

Reply:
huh, I witnessed the opposite behavior